boards/arm/ronoth_lodev/doc/index.rst

```diff
@@ -0,0 +1,183 @@
+.. _ronoth_lodev:
+
+Ronoth LoDev
+############
+
+========
+Overview
+========
+
+The Ronoth_ LoDev_ is a small open source board containing a `AcSIP S76S`_  SiP from AcSIP_.
+The `full LoDev design details`_ are available on on GitHub.  The LoDev_ board can be purchased
+from Ronoth_ or from CrowdSupply_.
+
+The S76S contains an `STMicro STM32L073RZ`_ MCU, a `Semtech SX1276`_ LoRaWAN transceiver,
+and a +20 dBm power amplifier.  Refer to `AcSIP S76S Product Information Brief`_ for details.
+
+Zephyr applications may use the **ronoth_lodev** configuration to run on this board.
+
+.. figure:: img/lodev.png
+     :width: 291px
+     :align: center
+     :height: 217px
+     :alt: Image of Ronoth LoDev open source development board containing S76S system on a chip
+
+     Ronoth LoDev
+
+`Board design files`_ are available on GitHub.
+
+Some non-Zephyr development has been `posted on MBed by Steve Osborn`_.
```

Comment:
This is odd to have in docs, I'd ask to remove it since its not relevant to the board from a hw point of view and not to the zephyr software.

Reply:
Well it's not MBED code, but it's code in the MBED repository, giving more information than the (unfortunately sketchy) manufacturer's datasheet, which honestly is very thin.  But hey, I'm not hung up on it, I will take the reference out.